components/net/http_loader.rs

```diff
@@ -601,6 +601,7 @@ fn prepare_devtools_request(request_id: String,
                             now: Tm,
                             connect_time: u64,
                             send_time: u64) -> ChromeToDevtoolsControlMsg {
+    debug!("preparing!!!");
```

Comment:
might want to clean these up 😄 

It's fine to leave debug statements around, but try to have them print something useful, e.g. `prepare_devtools_request: <url>` or something